.github/workflows/release-build-zip-file.yml

```
@@ -43,7 +43,7 @@ env:
 jobs:
   verify:
     name: 'Pre-build verification'
-    runs-on: ubuntu-latest
+    runs-on: ${{ fromJSON('{"group":"WooCommerce Release Checks"}') }}
```

actionlint (1.7.12)

[error] type of expression at "runs-on" must be string or array but found type "{group: string}"

(expression)